tests/lib/rss.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * RSS Library Test
+ */
+
+const fs = require('fs')
+const rss = require('../../lib/rss')
+const mockdate = require('mockdate')
+
+mockdate.set('2017-12-22T08:05:39.000Z')
```

Comment:
Make sure this mock only applies inside the test that needs it and cleaned/removed after that.